skrf/vectorFitting.py

```diff
@@ -1873,7 +1875,7 @@ def get_new_subckt_identifier():
             for n in range(self.network.nports):
                 str_input_nodes += 'p{} '.format(n + 1)
 
-            f.write('.SUBCKT s_equivalent {}\n'.format(str_input_nodes))
+            f.write('.SUBCKT {} {}\n'.format(fitted_model_name, str_input_nodes))
```

Comment:
```suggestion
            f.write(f'.SUBCKT {fitted_model_name} {str_input_nodes}\n')
```
should also work (?)

Reply:
Yes, I always forget that f-strings work with python3.6 already